### PR TITLE
Automate Revolutionary Innovation

### LINKED ISSUE #14702: Revolutionary Innovation needs automation
Unlike Innovation and Breaktrough Innovation, the level 15 option doesn't have a rule element

### PR DESCRIPTION
**Pull request by ammalagonc**
- Added automation (to the best of my abilities) and icons to the level 15 Inventor class features.

- Tighten up the RE for the Revolutionary Innovation class feature.

Please review when you get a moment.

Once this is good to go, this should close #14702.

Closes #14702

### CODE REVIEW

**Comment by ammalagonc** [comment]
Thanks to @stwlam fix, I could weed out the duplicates in the revolutionary-innovation.json. I tested the changes without the dupes and they work as intended. 

If someone can still review this one though that would be great.

File: packs/classfeatures/deadly-strike.json [hunk 1]
```diff
@@ -24,7 +24,41 @@
...
+            },
+            {
+                "definition": [
+                    "item:id:{actor|flags.pf2e.trackedItems.weaponInnovation}"
+                ],
+                "key": "AdjustStrike",
+                "mode": "add",
+                "predicate": [
+                    "item:trait:deadly-d10"
+                ],
```
Comment (TikaelSol) [inline packs/classfeatures/deadly-strike.json:57]:
```suggestion
                "predicate": [
                    {
                        "or": [
                            "item:trait:deadly-d8",
                            "item:trait:deadly-d10"
                        ]
                    }
                ],
```

File: packs/classfeatures/deadly-strike.json [hunk 2]
```diff
@@ -24,7 +24,41 @@
...
+        "rules": [
+            {
+                "definition": [
+                    "item:id:{actor|flags.pf2e.trackedItems.weaponInnovation}"
+                ],
+                "key": "AdjustStrike",
+                "mode": "add",
+                "property": "weapon-traits",
+                "value": "deadly-d8"
+            },
```
Comment (TikaelSol) [inline packs/classfeatures/deadly-strike.json:36]: This may need to have its priority pushed back

File: packs/classfeatures/deadly-strike.json [hunk 3]
```diff
@@ -24,7 +24,41 @@
...
+                    "item:id:{actor|flags.pf2e.trackedItems.weaponInnovation}"
+                ],
+                "key": "AdjustStrike",
+                "mode": "add",
+                "predicate": [
+                    "item:trait:deadly-d6"
+                ],
+                "property": "weapon-traits",
+                "value": "deadly-d10"
+            },
```
Comment (TikaelSol) [inline packs/classfeatures/deadly-strike.json:48]: Does this add both traits? If so we need another set of REs to remove the old trait (change the priority to 1 higher than the add RE has)

File: packs/classfeatures/extensible-weapon.json [hunk 4]
```diff
@@ -24,7 +24,17 @@
...
+            {
+                "definition": [
+                    "item:id:{actor|flags.pf2e.trackedItems.weaponInnovation}"
+                ],
+                "key": "AdjustStrike",
+                "mode": "add",
+                "property": "weapon-traits",
+                "value": "reach"
+            }
+        ],
```
Comment (TikaelSol) [inline packs/classfeatures/extensible-weapon.json:37]: We can also increase the reach by another 5 if the weapon has the reach trait. An AE like to add 5 to `system.attributes.reach.base`

File: packs/classfeatures/omnirange-stabilizers.json [hunk 5]
```diff
@@ -24,7 +24,17 @@
...
+            {
+                "definition": [
+                    "item:id:{actor|flags.pf2e.trackedItems.weaponInnovation}"
+                ],
+                "key": "AdjustStrike",
+                "mode": "add",
+                "property": "range-increment",
+                "value": 50
+            }
+        ],
```
Comment (TikaelSol) [inline packs/classfeatures/omnirange-stabilizers.json:37]: This needs to be predicated on not having the volley trait, and another RE to remove the volley trait if it exists (again, may need to play with priority)

File: packs/classfeatures/physical-protections.json [hunk 6]
```diff
@@ -24,7 +24,21 @@
...
+                "key": "Resistance",
+                "predicate": [
+                    "armor:id:{actor|flags.pf2e.innovationId}"
+                ],
+                "type": [
+                    "slashing",
+                    "piercing",
+                    "bludgeoning",
+                    "bleed"
+                ],
```
Comment (TikaelSol) [inline packs/classfeatures/physical-protections.json:38]:
```suggestion
                "type": [
                    "physical"
                ],
```
Don't know why Paizo spelled it out, all of them are just different types of physical damage.

File: packs/classfeatures/omnirange-stabilizers.json (hunk 5 above)
Comment (ammalagonc) [reply packs/classfeatures/omnirange-stabilizers.json]: Changes made - check when you get a moment

**Comment by ammalagonc** [comment]
OK, all of the changes have been added. Please review when you get a moment

**Pull request merged**